lib/archethic_web/live/chains/beacon_live.ex

```diff
@@ -0,0 +1,102 @@
+defmodule ArchEthicWeb.BeaconChainLive do
+  use ArchEthicWeb, :live_view
+
+  alias ArchEthic.BeaconChain
+  alias ArchEthic.BeaconChain.Summary, as: BeaconSummary
+  alias ArchEthic.BeaconChain.SummaryTimer
+  alias ArchEthic.Crypto
+  alias ArchEthic.Election
+  alias ArchEthic.P2P
+  alias ArchEthic.P2P.Node
+  alias ArchEthic.SelfRepair.Sync.BeaconSummaryHandler
+  alias ArchEthicWeb.ExplorerView
+  alias Phoenix.View
+
+  defp f2(date = %DateTime{}) do
+    Enum.map(BeaconChain.list_subsets(), fn subset ->
+        b_address = Crypto.derive_beacon_chain_address(subset,date,True)
+        node_list = P2P.authorized_nodes()
+        nodes = Election.beacon_storage_nodes(subset,date,node_list)
+        %Node{network_patch: patch} = P2P.get_node_info()
+
+          {b_address, nodes, patch}
+      end)
+      |> :lists.flatten()
+      |> Task.async_stream(
+        fn {address, nodes, patch} ->
+          BeaconSummaryHandler.download_summary(address, nodes, patch)
+        end,
+        on_timeout: :kill_task,
+        max_concurrency: 256
+      )
+      |> Enum.to_list()
+      |>:lists.flatten()
+      |> Stream.filter(&match?({:ok, {:ok, %BeaconSummary{}}}, &1))
+      |> Enum.map(fn {:ok, {:ok, %BeaconSummary{transaction_summaries: transaction_summaries}}} -> transaction_summaries end)
+      |>:lists.flatten()
+  end
+
+  def mount(_params, _session, socket) do
+    # Todo handle live transactions
+    # if connected?(socket) do
```

Comment:
It will be better to active realtime for beacon_summary transaction and update the pagination and the transaction list